docs/UserGuide.md

```diff
@@ -78,7 +78,7 @@ Examples of usage:
 `sales -a Izumi Tsubaki -m Gekkan Shoujo Nozaki-kun -q 1700 -p 12.90`
 
 ### Add Deadline to a Manga: `schedule`
-Adds a deadline to an existing manga. `-b` stands for by-date (in case you are wondering why it is not `-d`, which is used for deleting in other commands). 
+Adds a deadline to an existing manga. `-b` stands for by-date (in case you are wondering why it is not `-d`, which is used for deleting in other commands).
```

Comment:
Similar to a comment I got for the 'sales' data, I think you should also specify that the deadline is overwritten if ran again?

Reply:
Good point, I'll make that change.